demo/demo-dnt-helper.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/**
+ * Returns true or false based on whether doNotTrack is enabled. It also takes into account the
+ * anomalies, such as !bugzilla 887703, which effect versions of Fx 31 and lower. It also handles
+ * IE versions on Windows 7, 8 and 8.1, where the DNT implementation does not honor the spec.
+ * @see https://bugzilla.mozilla.org/show_bug.cgi?id=1217896 for more details
+ * @params {string} [dnt] - An optional mock doNotTrack string to ease unit testing.
+ * @params {string} [ua] - An optional mock userAgent string to ease unit testing.
+ * @returns {boolean} true if enabled else false
+ */
+var dntHelper = function (dnt, ua) {
```

Comment:
Suggestion: the demo should load the output of the package at `/dist/index.js` after running `npm run build`, otherwise it's hard to test that changes being made work correctly?

I think we can also avoid modifying the file for demo purposes. The demo page could instead load two JS files:

1. The compiles DNT helper JS file (no changes or alterations).
2. A demo-specific JS file that calls the DNT helper and updates the HTML.